taqueria-plugin-taquito/common.ts

```diff
@@ -0,0 +1,200 @@
+import {
+	getAccountPrivateKey,
+	getDefaultAccount,
+	getNetworkConfig,
+	getSandboxAccountConfig,
+	getSandboxAccountNames,
+	getSandboxConfig,
+	sendAsyncErr,
+	sendErr,
+	TAQ_ROOT_ACCOUNT,
+} from '@taqueria/node-sdk';
+import { Environment, NetworkConfig, RequestArgs, SandboxAccountConfig, SandboxConfig } from '@taqueria/node-sdk/types';
+import { importKey, InMemorySigner } from '@taquito/signer';
+import { TezosToolkit } from '@taquito/taquito';
+
+export interface OriginateOpts extends RequestArgs.ProxyRequestArgs {
+	contract: string;
+	storage: string;
+	alias?: string;
+	sender?: string;
+}
+
+export interface TransferOpts extends RequestArgs.ProxyRequestArgs {
+	contract: string;
+	tez?: string;
+	param?: string;
+	entrypoint?: string;
+	sender?: string;
+}
+
+export interface InstantiateAccountOpts extends RequestArgs.ProxyRequestArgs {
+}
+
+export interface FundOpts extends RequestArgs.ProxyRequestArgs {
+}
+
+// To be used for the main entrypoint of the plugin
+export type IntersectionOpts = OriginateOpts & TransferOpts & InstantiateAccountOpts & FundOpts;
+
+// To be used for common functions in this file
+type UnionOpts = OriginateOpts | TransferOpts | InstantiateAccountOpts | FundOpts;
+
+export const getFirstAccountAlias = (sandboxName: string, opts: UnionOpts) => {
+	const aliases = getSandboxAccountNames(opts)(sandboxName);
+	return aliases.shift();
```

Comment:
I think its a good idea to avoid mutation unless strictly needed. Otherwise it can create cognitive overhead. Thus, I'd suggest:
```
const [first, ..._] = getSandboxAccountNames(opts)(sandboxName);
return first
```

Reply:
Did not write this. But good idea. Will do